src/material/core/tokens/_token-definition-utils.scss

```diff
@@ -0,0 +1,266 @@
+@use 'sass:color';
```

Comment:
nit: maybe just `_token-definition.scss`?

Reply:
Yeah that sounds better - `util` naming is often code-smell